electron/legendary_utils/library.ts

```diff
@@ -209,10 +209,7 @@ class Library {
       dlcItemList,
       releaseInfo,
       categories,
```

Comment:
important bugfix, heroic does not start when the library includes unreal assets, because they do not have a cloudsavefolder